src/server/upload-and-scan/controller.js

```diff
@@ -81,6 +82,8 @@ const uploadController = {
       uploadDetails.pending = new Date().toISOString()
       await request.redis.storeUploadDetails(uploadId, uploadDetails)
 
+      await counter('upload-received')
```

Comment:
This is a counter once per entire multipart?
Do we want another per file uploaded? Or just per request like this one?
And do we want one earlier in this controller in case of errors?

Reply:
Basically do we want a counter for all request or just the valid ones. or both.

Reply:
My thinking here is this number should relate to the initialize counter so we can track how many uploads were requested vs actually made. 
Maybe a second counter where we track how many files were uploaded so we can tie it back to the clean/infected numbers?